src/Akeneo/Platform/Bundle/AnalyticsBundle/DataCollector/VersionDataCollector.php

```diff
@@ -81,4 +60,22 @@ protected function getServerVersion()
 
         return $version;
     }
+
+    private function appendResetData(array $collectedData): array
+    {
+        if ($this->featureFlags->isEnabled('reset_pim')) {
+            $resetData = $this->installStatusManager->getPimResetData();
+
+            if (null !== $resetData) {
+                $resetEvents = $resetData['reset_events'];
+                $collectedData = [
+                    ...$collectedData,
+                    'reset_event_count' => count($resetEvents),
+                    'last_reset_time' => end($resetEvents)['time'],
+                ];
+            }
+        }
+
+        return $collectedData;
```

Comment:
In my opinion we should return the number of reset even if the guy didn't reset her PIM for now
```suggestion
        if (!$this->featureFlags->isEnabled('reset_pim')) {
            return $collectedData;
        }
        
        $resetData = $this->installStatusManager->getPimResetData();
        $resetEvents = $resetData['reset_events'];
        $collectedData['reset_event_count'] = count($resetEvents);
        $collectedData['last_reset_time'] = empty($resetEvents) ? null : end($resetEvents)['time'],
```